### ISSUE TITLE
Rename the libs name

### ISSUE THREAD

**Issue by mengzhisuoliu** [openssl/openssl#20920]
MyTarget:
          rename the out lib, like libssl-1_1.lib->libssl-1_1-test.lib, libcrypto-1_1.lib->libcrypto-test-1_1.lib
Background:
    openssl 1.1.1t
    Windows
    Update the Configurations\descrip.mms.tmpl and Configurations\windows-makefile.tmpl files  From https://github.com/openssl/openssl/commit/58440f74e1bfc9e2eb7720fb635ec9b433c0aa8d
    But it seems the solution did not work on windows

I changed the code in the Configurations\windows-makefile.tmpl file ,it seems to be ok.
```
 sub lib {
     (my $lib = shift) =~ s/\.a$//;
     $lib .= '_static'
         if (defined $unified_info{sharednames}->{$lib});
	 #return $lib . $libext;
     return $lib . $shlibvariant . $libext;
 }
```

**Comment by mengzhisuoliu** [comment]
> 

sorry !i want to let `shlib_variant` setting to be applied on the (non-shared) or without (non-shared) this argument.
For Example:
Debug DLL
   libssl-1_1.lib  -> libssl-1_1d.lib
   libssl-1_1.dll -> libssl-1_1d.dll

Static Library
  libssl-1_1.lib -> libssl-1_1d.lib

Both that works well

**Comment by nhorman** [comment]
marking as inactive, will be closed at the completion of 3.4 dev without further input
